prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -289,6 +289,27 @@ periodics:
         requests:
           memory: 1Gi
           cpu: 400m
+- name: orphaned-dns-cleaner
+  cron: "45 */4 * * 1-5" # "At minute 45 past every 4th hour from Monday through Friday"
+  labels:
+    preset-sa-gke-kyma-integration: "true"
+    preset-gc-project-env: "true"
+  decorate: true
+  extra_refs:
+  - <<: *test_infra_ref
+    base_ref: master
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+      command:
+      - "bash"
+      args:
+      - "-c"
+      - "development/dns-cleanup.sh -project=${CLOUDSDK_CORE_PROJECT} -dnsZone='build-kyma' -ageInHours=2 -regions='europe-west3'"
```

Comment:
dnszone and region should be populated by preset that is also used by integration jobs